server/sonar-web/src/main/js/apps/webhooks/components/App.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import PageHeader from './PageHeader';
+import WebhooksList from './WebhooksList';
+import { searchWebhooks } from '../../../api/webhooks';
+import { LightComponent, Organization, Webhook } from '../../../app/types';
+import { translate } from '../../../helpers/l10n';
+
+interface Props {
+  component?: LightComponent;
+  organization: Organization | undefined;
+}
+
+interface State {
+  loading: boolean;
+  webhooks: Webhook[];
+}
+
+export default class App extends React.PureComponent<Props, State> {
+  mounted: boolean = false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Type boolean trivially inferred from a boolean literal, remove type annotation ([typescript:S3257](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3257))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWGTZEXDQeDOAQ7I7d5d&open=AWGTZEXDQeDOAQ7I7d5d)

[ISSUE_KEY]: AWGTZEXDQeDOAQ7I7d5d